### DISCUSSION TITLE
How to create a custom <Field> and handle it

### DISCUSSION THREAD

**Discussion by StErMi** [jaredpalmer/formik#2974] [Q&A]
Hi all,

I would like to create a switch field that allows me to turn it on and off but I didn't see a full example that allows me to bind a <Field> to a button and handle all the change events and validation. 

Could you please point me to an example?

**Comment by Dollique** [comment]
You can literally google 'custom formik component' and find lot's of examples.
If you use functional components there are also a lot of examples for custom hooks.

They may be able to help you better if you ask a specific issue you have.
